Source/WebCore/contentextensions/ContentRuleListResults.h

```diff
@@ -52,20 +52,14 @@ struct ContentRuleListResults {
                 || redirected
                 || !notifications.isEmpty();
         }
-
-        template<class Encoder> void encode(Encoder&) const;
-        template<class Decoder> static std::optional<Result> decode(Decoder&);
     };
     struct Summary {
         bool blockedLoad { false };
         bool madeHTTPS { false };
         bool blockedCookies { false };
         bool hasNotifications { false };
-        Vector<ContentExtensions::ModifyHeadersAction> modifyHeadersActions;
-        Vector<std::pair<ContentExtensions::RedirectAction, URL>> redirectActions;
-
-        template<class Encoder> void encode(Encoder&) const;
-        template<class Decoder> static std::optional<Summary> decode(Decoder&);
+        Vector<ContentExtensions::ModifyHeadersAction> modifyHeadersActions { };
```

Comment:
It would probably be worth adding a comment saying that these two are not serialized because _WKContentRuleListAction doesn't need them.